app/views/includes/scripts.njk

```diff
@@ -51,6 +51,10 @@
       analyticsTrackConfirmClick().init('{{analytics.analyticsId}}', '{{analytics.type}}', '{{analytics.paymentProvider}}', '{{analytics.amount}}', '{{hitPage}}');
     }
   });
+  {% if allowWebPayments %}
+    window.googlePayGatewayMerchantID = '{{ googlePayGatewayMerchantID }}';
+    window.googlePayMerchantID = '{{ googlePayMerchantID }}';
```

Comment:
i guess we always get these google pay specific things even if the user only has apple pay enabled on the device?

Reply:
Yeah that’s fine we dump loadsa bits down here just so that the client side script can grab hold of them if it needs them.